data-generator/data/GenerateData.py

```diff
@@ -42,10 +34,141 @@ def write_row(writer):
+
+
+def nationality(fake):
+    return fake.random.choice(["United Kingdom", "Italian", "German"])
+
+
+def ethnic_origin(fake):
+    return fake.random.choice(["Mixed or Multiple ethnic groups", "White", "Asian or Asian British",
+                               "Black, African, Caribbean or Black British", "Other ethnic group"])
+
+
+def religion(fake):
+    return fake.random.choice(
+        ["Polytheism", "Monotheism", "Atheism", "Animism", "Totemism"])
+
+
+def sexual_orientation(fake):
+    return fake.random.choice(["Heterosexual", "Homosexual", "German"])
+
+
+def employment_status(fake):
+    return fake.random.choice(["worker", "employee", "self-employed and contractor",
+                               "director", "office holder"])
+
+
+def date(fake):
+    return fake.date(pattern="%d-%m-%Y", end_datetime=datetime.date(2000, 1, 1))
+
+
+if __name__ == '__main__':
+    no_of_records = int(os.getenv("NO_OF_RECORDS", 2))
+    headers = ["PIDNo", "Full Name", "Name", "First Name", "Last Name", "Appointed (ERP) Office Name",
+               "Appointed (ERP) Office Town", "Appointed (ERP) Office Postcode", "Appointed (ERP) Office GOR",
+               "OfficeLat", "OfficeLong", "Steps Code", "Contract Type", "Employee Group", "Perm/Temp", "FTA End Date",
+               "Grade", "HMRC Grade", "SiP Grade", "SIP Grade (FTA)", "Profession", "Temp Promotion", "Start of TP",
+               "End of TP", "Temp Promotion Reason", "Temp Promotion Grade", "Date of Birth", "Age", "Entry Date",
+               "Initial start date", "Date to CS", "Years in CS", "Organizational Unit", "Organizational Unit Text",
+               "Pay Scale Area", "Pay Scale Group", "Pay Scale Level", "Pay Scale Type", "Payroll Area",
+               "Personnel Subarea", "Position", "Position Text", "Weekly Hrs", "Work schedule", "Work schedule rule",
+               "Cost Code", "Cost Centre Name", "Admin/Prog", "Directorate", "Line of Business",
+               "Sub-Directorate Group", "Line Manager PID", "Line Manager Name", "Line Manager Grade",
+               "Line Manager GSI Address", "Chief Position", "FT/PT", "Gender", "Marital status", "Ethnic Origin",
+               "Ethnicity Status", "Ethnicity Updated", "National Identity", "Nationality", "Disability",
+               "Disability Status", "Disabilty Updated", "Sexual Orientation Updated", "Sexual Orientation",
+               "Sexual Orientation Category", "Religion Or Belief Updated", "Religion Or Belief",
+               "Religion Or Belief Category", "Employment Status", "Partial retirement date", "FTE", "Absence Start",
+               "Half Pay Start", "Pension Rate Start", "Pension Rate End", "No Pay Date", "Absence End", "Type of Pay",
+               "Redeployment Status", "PreSurplus Start Date", "Redeployment Start Date", "Surplus start Date",
+               "BP Pack Dir", "BU Breakdown", "Distribution BU", "SIP", "GSI Address", "Annual salary", "NINO"]
```

Comment:
Please change this to one per line and sort alphabetically (see below - disregard the alpha sort for now if you'd like).